Source/SwiftLintCore/Models/Example.swift

```diff
@@ -120,7 +120,7 @@ extension Example {
     }
 
     /// Makes the current example focused. This is for debugging purposes only.
-    func focused() -> Example { // swiftlint:disable:this unused_declaration
+    public func focused() -> Example { // swiftlint:disable:this unused_declaration
```

Comment:
Could you please revert this? I'm making it public in a separate PR: https://github.com/realm/SwiftLint/pull/4959